pages/runbooks/authorize_ssh_key.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Authorize SSH key at runtime
+layout: page
+parent: Runbooks
+---
+
+# Authorize SSH key at runtime
+
+To add an authorized SSH key at runtime, connect to the device's console with
+a serial cable or OTA platform. At the IEx prompt, replace your public key in
+the following snippet and paste it into IEx. You should now be able to SSH into
+the device with the new key.
+
+```elixir
+key = "<public key>"
+Application.put_env(:nerves_ssh, :authorized_keys, [key])
+Application.stop(:nerves_ssh); Application.ensure_all_started(:nerves_ssh)
+```
+
+The authorized keys will persist after a reboot.
```

Comment:
:question: It don't need `persistent: true` opt?

Reply:
👍 Well, it *should* need that now that you mention it...

Looks like there may be something going on with `nerves_ssh`.

Reply:
✅ Updated